docs/RationaleHW-SV.md

```diff
@@ -388,6 +388,32 @@ still many things in SystemVerilog that we cannot currently express in the SV
 dialect, this design makes it easy to incrementally build out new capabilities
 over time.
 
+#### Verbatim op
+The verbatim operation produces a typed value expressed by a string of
+SystemVerilog.  This can be used to access macros and other values that are
+only sensible as Verilog text. There are three kinds of verbatim operation, 
+ 1. VerbatimOp(`verbatim`, the statement form
```

Comment:
nit, but please include the dialect prefix for clarity `sv.verbatim` instead of just verbatim